.github/workflows/release.yml

```
@@ -45,8 +48,16 @@ jobs:
       with:
         toolchain: stable
         target: ${{ matrix.target }}
+    - name: Download cross-compilation tool (linux-aarch64)
+      if: matrix.target == 'aarch64-unknown-linux-musl'
+      run: wget -c https://github.com/cross-rs/cross/releases/download/v0.2.5/cross-x86_64-unknown-linux-gnu.tar.gz -O - | tar -xz
     - name: Build release binary
-      run: cargo build --target ${{ matrix.target }} --verbose --release --features packaging,vendored-openssl
+      run: |
+        if [ "${{ matrix.target }}" = "aarch64-unknown-linux-musl" ]; then
+          ./cross build --target ${{ matrix.target }} --verbose --release --features packaging,vendored-openssl
+        else
+          cargo build --target ${{ matrix.target }} --verbose --release --features packaging,vendored-openssl
+        fi
```

Minor suggestion that I think reads a tiny bit better.

```suggestion
      run: |
        CARGO_CMD=cargo
        if [ "${{ matrix.target }}" = "aarch64-unknown-linux-musl" ]; then
          CARGO_CMD=$PWD/cross
        fi
        $CARGO_CMD build --target ${{ matrix.target }} --verbose --release --features packaging,vendored-openssl
```

---

done :+1: